src/NativeScript/TNSRuntime+Inspector.mm

```diff
@@ -49,14 +50,19 @@ @interface TNSRuntimeInspector ()
 
 - (instancetype)initWithRuntime:(TNSRuntime*)runtime
                  messageHandler:(TNSRuntimeInspectorMessageHandler)messageHandler;
+- (void)setup;
 
 @end
 
 @implementation TNSRuntime (Inspector)
 
 - (TNSRuntimeInspector*)attachInspectorWithHandler:(TNSRuntimeInspectorMessageHandler)messageHandler {
-    return [[[TNSRuntimeInspector alloc] initWithRuntime:self
-                                          messageHandler:messageHandler] autorelease];
+    TNSRuntimeInspector* runtimeInspector = [[[TNSRuntimeInspector alloc] initWithRuntime:self
+                                                                           messageHandler:messageHandler] autorelease];
+
+    [runtimeInspector setup];
+
+    return runtimeInspector;
```

Comment:
I think it's better to move the `autorelease` call here, because this is where the function returns.